src/core_plugins/vega/public/vega_view/vega_base_view.js

```diff
@@ -329,6 +330,43 @@ export class VegaBaseView {
     return { from, to, mode };
   }
 
+  /**
+   * Set global debug variable to simplify vega debugging in console. Show info message first time
+   */
+  setDebugValues(view, spec, vlspec) {
+    if (!window) {
+      return;
+    }
+
+    if (window.VEGA_DEBUG === undefined && console) {
+      console.log('%cWelcome to Kibana Vega Plugin!', 'font-size: 16px; font-weight: bold;');
+      console.log('You can enable debugging by putting `%debug%: chooseAnyIdHere` into the top level of our spec.');
```

Comment:
What about adding `%debug%: chooseAnyIdHere` into the `default.spec.hjson`, commented out, so it's easier instead of copying it from the console the first time?

Reply:
Should we actually trying to promote it that much? It's a temporary feature until we'll have some proper inspector panel for those. Also I think vega is still used by people maybe not even familiar with the browser dev tools, so I rather wouldn't try to point those towards opening their browser dev tools. But my main worry would be, that a lot of people will then modify their spec and have that as a comment in there forever, even after we created the inspector panel. So we kind of have "false" documentation stored in a lot of specs.